src/sync_backend_clients/dropbox_sync_backend_client.js

```diff
@@ -15,7 +16,7 @@ export const filterAndSortDirectoryListing = (listing) => {
     // Show all folders
     if (file['.tag'] === 'folder') return true;
     // Filter out all non-org files
-    return file.name.match(/org$|org_archive$/);
+    return file.name.match(orgFileExtensions);
```

Comment:
OK, I understand – we already have this file filter...

Reply:
Yes, and I've now fixed it to include the `.` :-)